### PR TITLE
enterprise: Added developerbox template materials

### PR DESCRIPTION
**Pull request by sdrobertw**
All developerbox template materials should be populated prior to product launch. This includes links to internal and external resources.

Signed-off-by: Robert Wolff <robert.wolff@linaro.org>

### CODE REVIEW

**Comment by daniel-thompson** [comment]
@shovanuk 

I tried last week but couldn't make head or tail of the link checker output. I think Robert also had a go but couldn't work out what was causing the last few problems.

**Comment by sdrobertw** [comment]
@daniel-thompson I thought you reverted all of your changes to the last PR and incidentally closed it and opened a new one to test link issues...

I am still trying to figure out this link thing, and hope to have it settled asap...

**Comment by daniel-thompson** [comment]
Nope. The PR I submitted was tidied up; I thought you had derived this PR from my one.

Either way... if we can land this one then fixing the typos in a new PR should be easy because changes like that won't anger the link checker.

**Comment by ric96** [comment]
the build should be fine once merged,
for ex:
the link check is failing at https://github.com/96boards/documentation/tree/master/enterprise/developerbox/support/README.md
giving a 404 error, and the page doesn't exist as the PR is not merged.
Same for the other errors

**Comment by sdrobertw** [comment]
@ric96 I don't know how we can merge without passing the link test...

Thoughts @shovanuk @kylekirkby @pcolmer ?

**Comment by sdrobertw** [comment]
@shovanuk @kylekirkby @pcolmer okay... I removed all of the redirects and it is still saying the links are broken. This is starting to cause issues for me. I have 5 boards in pipeline and do not want to send PR for them until I have this figured out.

Can we please schedule something so I can be shown what I am doing wrong? No one can seem to get rid of these broken links.

**Comment by pcolmer** [comment]
@sdrobertw Looking at the built pages, it looks like there is a problem with the finished page, rather than your specific contribution. The page has additional material like this:

    <a type="button" href="https://github.com/96boards/documentation/tree/master/enterprise/developerbox/additional-docs/README.md" class="btn btn-default github_control_btn btn-buy" id="edit_on_github"> <span class="top-text">Edit</span> <span class="bottom-text">on Github</span> </a>

In other words, someone has made a decision to be "helpful" by providing a button that is supposed to allow the page to be edited directly on GitHub. There is where the link is coming from. I'm not sure how helpful this approach is (apart from the broken link aspect) because any changes need a forked copy of the repo in the first instance.

**Comment by daniel-thompson** [comment]
@sdrobertw @pcolmer 

Thanks for the analysis Philip.

The links are certainly useful when generating the production site... however having them in the template used for the CI loop is broken. Unfortunately bamboo is rather too opaque for me to have developed any insight into how we could substitute templates for the CI build.

Also doesn't this mean that the current CI machinery currently prevents the introductions of any new file?
An error of this magnitude strikes me as large enough justify disabling the CI loop until it is fixed!

**Comment by pcolmer** [comment]
@daniel-thompson I can certainly disable the CI tests, in order to allow merges to continue, but that will only push the problem down the pipe since the link checks are also performed as part of the build process and the system will refuse to upload changes that result in broken links.

I think that the problem with the "edit" links is that the pages don't yet exist and that is why the links are "broken". I honestly don't know how to address that in the build process other than to just ignore any and all references to github.com, which seems a bit of a sledgehammer to crack this nut.

I'll turn off the checks in both github and Bamboo in order to get the pages onto the system.

**Comment by daniel-thompson** [comment]
@pcolmer 

To be honest I agree entirely about pushing the problem down the pipe... and I also saw the work @kylekirkby did chasing down the last of the broken links so it could be turned on in the first place.

I also agree that excluding github links from the checker that is a bit of a sledgehammer... however if it is relatively quick to implement I guess it might be better than the wrecking ball of turning off all CI checks!

**Comment by daniel-thompson** [comment]
PS Can the CI link checks be made advisory (so a manual override is possible) rather than blocking?

**Comment by pcolmer** [comment]
This PR can be merged once there is an approving review. I've turned off link checking for now.

**Comment by pcolmer** [comment]
OK - please note that this PR is now failing because of a missing file. You've got this directory structure:

`/documentation/enterprise/developerbox/getting-started/basic-kit`

with a page under "basic-kit". However, you haven't provided a page under "getting-started" and the breadcrumb code requires there to be a page at each level. Until a "getting-started" page exists, this PR will continue to fail.

P.S. I've trained the link checker to ignore specifically the "edit on GitHub" links, with a huge thanks to @kylekirkby for making it a trivial change :)

**Comment by sdrobertw** [comment]
@daniel-thompson can you please approve :)

**Comment by sdrobertw** [comment]
Link and build reqs are gone? Did we break something? @shovanuk @kylekirkby @pcolmer 

>.< lol

**Comment by daniel-thompson** [comment]
@sdrobertw: Er... sure... how?
